### PR TITLE
Add basic webpack support for sass compilation

### PR DESCRIPTION
**Pull request by slomo**
- remove compress for base style
- add loaders to for sass, css and style
- adopt bootstrap paths

### CODE REVIEW

**Comment by vellip** [comment]
@slomo check this out!

File: euth_wagtail/static/scss/all.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "../icons/fontcustom/fontcustom.css";
+/* @import "../icons/fontcustom/fontcustom.css"; */
```
Comment (slomo) [inline euth_wagtail/static/scss/all.scss]: Not sure about this. Probably we might still need it.

File: euth_wagtail/static/scss/core/_bootstrap.scss [hunk 2]
```diff
@@ -11,7 +11,7 @@
 // Reset and dependencies
 @import "bootstrap/normalize";
 @import "bootstrap/print";
-@import "bootstrap/glyphicons";
+// @import "bootstrap/glyphicons";
```
Comment (slomo) [inline euth_wagtail/static/scss/core/_bootstrap.scss]: Neither sure what happens here.

**Comment by slomo** [comment]
How does this work? You are using the extract loader to exclude the style loader from the bundle, but the extract css is not included anywhere?

File: euth_wagtail/static/scss/core/_bootstrap.scss (hunk 2 above)
Comment (vellip) [inline euth_wagtail/static/scss/core/_bootstrap.scss]: This is fine, it's actually not needed from what I've found.

**Comment by slomo** [comment]
LGTM @MagdaN do you want to check this or should I merge it?

**Pull request merged**